packages/vanilla/src/sass/components/_pagination.scss

```diff
@@ -1,153 +1,67 @@
-//@use '../base/colors' as *;
-
-// TODO - needs color tokens in place of hex codes
-// TechDebt: Why is regular list CSS here?
-
-/* Potential CBP List styling */
-.cbp-list {
-  display: flex;
-  flex-wrap: wrap;
-  list-style: none;
-}
-
-.cbp-list > li {
-  border-bottom: 1px solid #F1F3F6; // Token?
-  flex: 1 0 100%;
-  max-width: 100%;
-}
-
-/* Gives space above to all phone numbers except first one in accordion */
-// TechDebt: is this comment relevant or copy-paste?
-.cbp-list > li:not(:first-child) { 
-  margin-top: 16px;
-}
-
-.cbp-list > li div {
-  display: flex;
-  flex-direction: column;
-  flex-wrap: wrap;
-  justify-content: flex-start;
-  padding: 0;
-}
-
-.cbp-list__info {
-  list-style: none;
-  padding-left: unset;
-}
-
-.cbp-list > li div button {
-  margin-bottom: 16px;
-  width: fit-content;
-}
-
-.cbp-page-list {
-  align-items: center;
-  display: flex;
-  flex: 0 1 100%;
-  justify-content: flex-start;
-  
-  & > select {
-    all: revert;
-    padding: 8px;
-  }
-}
-
-.cbp-pagination {
-  margin-left: auto;
-}
+@use './button/base';
+@use './button/secondary';
+@use './select';
 
 .cbp-pagination {
   align-items: center;
   display: flex;
   justify-content: space-between;
```

Comment:
I would add `flex-wrap: wrap` to this. They are overlapping at mobile size even though you've hidden the page size dropdown.